src/http.ts

```diff
@@ -12,40 +12,37 @@ export async function request(
   url: string,
   method: Method,
   data = '{}'
-): Promise<[number, Object]> {
-  switch (method.toUpperCase()) {
-    case 'POST':
-      return post(url, data)
-    case 'GET':
-      return get(url)
+): Promise<[number, Object, Object]> {
+  try {
+    const response = await axios.request({
+      url,
+      method,
+      data,
+      headers: {'Content-Type': 'application/json'}
```

Comment:
@titoine54 here's where you could pass the input headers 👆🏽

Right now I hardcode the `Content-Type` for GraphQL queries, so once we allow the user to pass custom headers, we can leave this one as default.